app/services/lifetime_usages/flag_refresh_from_subscription_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module LifetimeUsages
+  class FlagRefreshFromSubscriptionService < BaseService
+    def initialize(subscription:)
+      @subscription = subscription
+      super
+    end
+
+    def call
+      return result unless subscription.active?
+
+      lifetime_usage = subscription.lifetime_usage
+      lifetime_usage ||= subscription.build_lifetime_usage(organization: subscription.organization)
```

Comment:
I'm wondering, this will mean we'll track the usage for all active subscriptions. Should we try and limit it only to those that have thresholds defined?

Reply:
@nudded To avoid overloading the workers and DB, maybe we should indeed check for the presence of thresholds.
Updating it!